lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -1,16 +1,26 @@
+require "cc/engine/analyzers/parser_error"
+
 module CC
   module Engine
     module Analyzers
       class Base
+        RESCUABLE_ERRORS = [
```

Comment:
I'd try to match the behavior of Flay run on the command line then. If the Flay catches the timeout error, logs a skip, and continues on, I think we should do the same.